cluster/HDdistances.py

```diff
@@ -0,0 +1,71 @@
+
+# This file provides functionalities for High dimensionality problems but also for low dimensionality problems
+#   - New Distance computation
+#   - SSE metric computation for assist the computation of the optimal number of clusters
+#
+# Authors:
+# Jose Javier Garcia Aranda
+# Juan Ramos Diaz
+
+
+
+#from cluster import KMeansClustering
+#import KMeansClustering
+#import ClusteringError
+import util
+import time
+import datetime
+
+import random
+
+def HDdistItems(profile1,profile2):
+    #Distance function, this distance between two profiles is based on:
+    #For each keyword of user A, if the keyword is not  present in user B , then the distance for this keyword is the weight in the user A.
+    #If the keyword exists in both users, the weights are compared and the distance is the absolute difference
+    len1=len(profile1)/2
+    len2=len(profile2)/2
+    total_len=len1+len2 #this value usually is 20
```

Comment:
If you look at this function in isolation (without any other context), then this comment is not correct and should be removed.

Reply:
i agree